Technical-Steering-Committee/best-practices/contrinuous-integration.md

```diff
@@ -0,0 +1,23 @@
+# Continuous Integration
+
+## Prerequisites
+All Zowe repositories containing source code that requires compilation, packaging, testing or other type of verification, should define a continuos integration process that uses FLOSS automation tools, free to acquire and use for any purposes. It is recommended to choose tools already popular within Zowe, such as Gradle, npm, GitHub Actions, etc. For specific needs, it is possible to select alternatives, given they provide unique and necessary advantages.
+
+## Compilation
+If a Zowe repository requires source code compilation, this repository should include compilers, scripts and tools that automate the process, handle dependencies and produce project artifacts.
+
+## Packaging
+If a Zowe module is intended to be delivered as a deployable installation packed, its corresponding repositories should contain scripts and tools that group the necessary  project artifacts included or produced by compilation into a single or multiple packages targeted for their deployment environments.
```

Comment:
```suggestion
If a Zowe module is intended to be delivered as a deployable installation package, its corresponding repositories should contain scripts and tools that group the necessary  project artifacts included or produced by compilation into a single or multiple packages targeted for their deployment environments.
```

Reply:
Maybe a shorter version would be sufficient: 
```suggestion
If a Zowe module is intended to be delivered as a deployable installation packed, its corresponding repositories should contain scripts and tools that when executed/used produce automatically these artifacts.
```

Reply:
Resolved in the latest commit.